src/test/groovy/com/palantir/gradle/revapi/RevapiSpec.groovy

```diff
@@ -311,6 +311,29 @@ class RevapiSpec extends IntegrationSpec {
         assert standardError.contains('willBeRemoved')
     }
 
+    def 'if there are no published versions of the library at all, dont error out'() {
+        when:
+        buildFile << """
+            apply plugin: '${TestConstants.PLUGIN_NAME}'
+            apply plugin: 'java-library'
+            
+            repositories {
+                mavenCentral()
+            }
+            
+            revapi {
+                oldGroup = 'does.not'
+                oldName = 'exist'
+                oldVersion = '1.0.0'
```

Comment:
how does this get set normally, if you don't have a version?
would it be set to something or empty? if empty, we should test that code path

Reply:
Normally it gets set to the last 3 git tags (it's a list underneath). The "no git tags" case is tested by [this test](https://github.com/palantir/gradle-revapi/blob/bbe721711ddfc44b92896c0232e792760a1d9ae5/src/test/groovy/com/palantir/gradle/revapi/RevapiSpec.groovy#L234) ([PR for that change](https://github.com/palantir/gradle-revapi/pull/120)).

Reply:
ah ok then